src/Util/Log/JUnit.php

```diff
@@ -229,6 +229,7 @@ public function startTestSuite(TestSuite $suite)
     {
         $testSuite = $this->document->createElement('testsuite');
         $testSuite->setAttribute('name', $suite->getName());
+        $testSuite->setAttribute('package', $suite->getName());
```

Comment:
Not sure what package name should be in this context though. I took the approach from https://www.npmjs.com/package/karma-junit-reporter which does call it suite

Perhaps better to introduce a new variable for this particular string?